testsuite/integration/basic/pom.xml

```diff
@@ -1432,13 +1432,31 @@
             </plugin>
             <plugin>
                 <groupId>org.wildfly.plugins</groupId>
-                <artifactId>wildfly-jar-maven-plugin</artifactId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>${version.org.wildfly.plugin}</version>
+                <configuration>
+                    <overwrite-provisioned-server>true</overwrite-provisioned-server>
+                    <record-provisioning-state>false</record-provisioning-state>
+                    <log-provisioning-time>${galleon.log.time}</log-provisioning-time>
+                    <offline-provisioning>${galleon.offline}</offline-provisioning>
+                    <galleon-options>
+                        <jboss-maven-dist/>
+                        <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                        <optional-packages>passive+</optional-packages>
+                    </galleon-options>
+                    <bootableJar>true</bootableJar>
+                    <skipDeployment>true</skipDeployment>
```

Comment:
This is somewhat related to the `wildfly-maven-plugin` more than this change. I assume this is to make this a hallow JAR. If that is the case, I think we need to update the description on the plugins documentation.

Reply:
Good point, I logged https://issues.redhat.com/browse/WFMP-270
BTW, you can create an Hollow JAR without this property, but you will get a Warning (Deployment not found). It is cleaner to generalize that and document it (as you suggested).